### PR TITLE
My Job Page: UI

### PR DESCRIPTION
**Pull request by diVladimirov**
NonPost
Posts

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -22,6 +23,11 @@ const SignIn = lazy(() => import('pages/SigninPage'));
 
 const ForgotPassword = lazy(() => import('components/forgotPassword/forgotPassword'));
 
+const PostJobPage = lazy(() => import('pages/PostJobPage' /* webpackChunkName: "post-job-page" */));
+const JobDescriptionPage = lazy(
+	() => import('pages/JobDescriptionPage' /* webpackChunkName: "job-description-page" */),
+);
```
Comment (lyudad) [inline src/App.tsx]: delete all comments

File: src/components/PostJobPage/PostJobLayout.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	height: 40px;
+	overflow: hidden;
+`;
+
+export const DateStyled = styled.div`
+	width: 200px;
+	height: 35px;
+	display: flex;
+	align-items: center;
+	background-color: gray;
```
Comment (lyudad) [inline src/components/PostJobPage/PostJobLayout.styles.tsx]: to constants

File: src/components/PostJobPage/PostJobLayout.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	color: white;
```
Comment (lyudad) [inline src/components/PostJobPage/PostJobLayout.styles.tsx]: same, all colors

File: src/App.tsx (hunk 1 above)
Comment (diVladimirov) [reply src/App.tsx]: fixed it

File: src/components/PostJobPage/PostJobLayout.styles.tsx (hunk 2 above)
Comment (diVladimirov) [reply src/components/PostJobPage/PostJobLayout.styles.tsx]: add constants in theme by styled components

File: src/components/PostJobPage/PostJobLayout.styles.tsx (hunk 3 above)
Comment (diVladimirov) [reply src/components/PostJobPage/PostJobLayout.styles.tsx]: add constants in theme by styled components

**Pull request merged**